.flake8

```diff
@@ -0,0 +1,11 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.
+
+[flake8]
+filename = *.py,
+exclude = __init__.py, *.egg, build, docs, .git
+force-check = True
```

Comment:
The `force-check` option is only needed for Cython libraries that want to enable flake8 support. We can drop it here since there's no Cython. See: https://github.com/kmaehashi/flake8-force

Reply:
```suggestion
```